kinetic/distribution.yaml

```diff
@@ -10495,7 +10495,19 @@ repositories:
   realsense2_camera:
     doc:
       type: git
-      url: https://github.com/intel-ros/realsense.git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: development
+    release:
+      packages:
+      - realsense2_camera
+      - realsense2_description
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/IntelRealSense/realsense-ros-release.git
+      version: 2.2.6-1
```

Comment:
It looks like this version should be [`2.2.6-2`](https://github.com/IntelRealSense/realsense-ros-release/blob/88de910e658188de3170b8f96d7844683f61a71e/README.md#realsense2_camera-kinetic---226-2)

Reply:
You are right. Done.